osc-server/src/main/java/org/osc/core/broker/service/tasks/conformance/securitygroup/MarkSecurityGroupMemberDeleteTask.java

```diff
@@ -21,12 +21,16 @@
 import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
 import org.osc.core.broker.service.persistence.OSCEntityManager;
 import org.osc.core.broker.service.tasks.TransactionalTask;
+import org.osc.core.broker.util.log.LogProvider;
 import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
 
 @Component(service=MarkSecurityGroupMemberDeleteTask.class)
 public class MarkSecurityGroupMemberDeleteTask extends TransactionalTask {
 
-    @Override
+   private static final Logger LOG = LogProvider.getLogger(MarkSecurityGroupMemberDeleteTask.class);
```

Comment:
please make sure we dont have unused fields. Eclipse should be providing warnings...please clean them up if they are related to your code change

Reply:
I pulled in these changes with rebase.

Reply:
Maybe it was merge issue. This line is not in the master branch.